### ISSUE TITLE
cosIntegral - wrong values in parametric plot

### ISSUE THREAD

**Issue by axkr** [paulmasson/mathcell#25]
cosIntegral  - parametric plot isn't continously.

```
<?xml version="1.0" encoding="UTF-8"?>

<!DOCTYPE html PUBLIC
  "-//W3C//DTD XHTML 1.1 plus MathML 2.0 plus SVG 1.1//EN"
  "http://www.w3.org/2002/04/xhtml-math-svg/xhtml-math-svg.dtd">

<html xmlns="http://www.w3.org/1999/xhtml" style="width: 100%; height: 100%; margin: 0; padding: 0">
<head>
<meta charset="utf-8">
<title>MathCell</title>
</head>

<body style="width: 100%; height: 100%; margin: 0; padding: 0">

<script src="https://cdn.jsdelivr.net/gh/paulmasson/math@1.4.5/build/math.js"></script>
<script src="https://cdn.jsdelivr.net/gh/paulmasson/mathcell@1.9.2/build/mathcell.js"></script>
<script src="https://cdn.jsdelivr.net/gh/mathjax/MathJax@2.7.5/MathJax.js?config=TeX-AMS_HTML"></script>
<div class="mathcell" style="display: flex; width: 100%; height: 100%; margin: 0;  padding: .25in .5in .5in .5in; flex-direction: column; overflow: hidden">
<script>

var parent = document.currentScript.parentNode;

var id = generateId();
parent.id = id;

MathCell( id, [  ] );

parent.update = function( id ) {


function z1(z) { try { return  cosIntegral(pow(z,2));}catch(e){return complex(Number.NaN);} }

var p1 = parametric( (re,im) => [ re, im, z1(complex(re,im)) ], [-2.0, 2.0], [-2.0, 2.0], { complexFunction: 'abs', colormap: 'complexArgument' } );

  var config = { type: 'threejs', aspectRatio: [1.0,1,1], zMin: -2, zMax: 15 };
  var data = [p1];
evaluate( id, data, config );

}

parent.update( id );

</script>
</div>

</body>
</html>
```

![image](https://user-images.githubusercontent.com/3109793/127059298-18270d00-c0ea-4dba-9dbf-56a8894f846f.png)

**Comment by paulmasson** [comment]
@axkr this plot should look better now but still has issues on the unit circle as noted [here](https://github.com/paulmasson/mathcell/issues/24#issuecomment-1086972915)

**Comment by paulmasson** [comment]
@axkr fixed issues in the underlying expIntegralEi around the origin, so this should look much better. Included in release [1.4.8](https://github.com/paulmasson/math/releases/tag/1.4.8).

Feel free to reopen if there are additional issues. Thanks for the input!
